### PR TITLE
Story Card API contract

### LINKED ISSUE #71: [Story Card] API Contract
- It is a collection of [Tasks](https://github.com/Real-Dev-Squad/website-api-contracts/tree/main/tasks)
- Containing the following data
```js
{
  title: String,
  description: String,
  status: Enum,
  tasks: [
    <TaskId>
    <TaskId>
  ],
  featureOwner: <UserId>,
  backendEngineer: <UserId>, 
  frontendEngineer: <UserId>,
  StartedOn: Epoch,
  EndsOn: Epoch, // Due date 
}
```

### PR DESCRIPTION
**Pull request by shubham-y**
Closes #71

### CODE REVIEW

File: story/README.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    "status": "Enum",
+    "tasks": [
+        "<TaskId>",
+        "<TaskId>"
+    ],
+    "featureOwner": "<UserId>",
+    "backendEngineer": "<UserId>",
+    "frontendEngineer": "<UserId>",
+    "StartedOn": "Epoch",
+    "EndsOn": "Epoch"
```
Comment (whyDontI) [inline story/README.md:18]: Make JSON keys `camelCase`

File: story/README.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
+# Story
+
+## Story object
+
+```json
+{
+    "title": "String",
+    "description": "String",
+    "status": "Enum",
```
Comment (whyDontI) [inline story/README.md:9]: Please list `Enum` values

**Review by whyDontI — CHANGES_REQUESTED** [review]
Requesting some small changes, also please indent all the JSON with 4 space indentation.

File: story/README.md (hunk 2 above)
Comment (shubham-y) [reply story/README.md]: Added status `Enum` values same as task status

File: story/README.md (hunk 1 above)
Comment (shubham-y) [reply story/README.md]: Changes done

**Review by whyDontI — APPROVED** [review]
LGTM 👍🏽 🥳

**Pull request merged**